src/shared/edit-util.c

```diff
@@ -189,18 +257,37 @@ int run_editor(const EditFile *files) {
                                 args[i] = editor_args[i];
                 }
 
-                if (files[0].path && files[0].line > 1 && !files[1].path) {
+                if (context->n_files == 1 && context->files[0].line > 1) {
                         /* If editing a single file only, use the +LINE syntax to put cursor on the right line */
-                        if (asprintf(args + i, "+%u", files[0].line) < 0) {
+<<<<<<< HEAD
+                        if (asprintf(args + i, "+%u", context->files[0].line) < 0) {
                                 (void) log_oom();
                                 _exit(EXIT_FAILURE);
+=======
+                        char l[3];
+
+                        xsprintf(l, "+%u", context->files[0].line);
+
+                        r = strv_extend(&args, l);
+                        if (r < 0)
+                                goto oom;
+
+                        r = strv_extend(&args, context->files[0].temp);
+                        if (r < 0)
+                                goto oom;
+                } else
+                        EDIT_FILES_FOREACH(i, context) {
+                                r = strv_extend(&args, i->temp);
+                                if (r < 0)
+                                        goto oom;
```

Comment:
Nah, in the 4th commit, `args` is allocated with `newa()`. Hence, we cannot use `strv_extend()`.

Reply:
I think it does strdup internally? But I'm not familiar with `alloca`...